testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -429,7 +429,11 @@ def ens2(self, ens2_template):
             ens2_template.filename,
             ens2_template.trajectory.timeseries(order='fac'),
             format=mda.coordinates.memory.MemoryReader)
-    
+
+    @pytest.fixture()
+    def empty_cluster(self):
```

Comment:
Agreed with @tylerjereddy (https://github.com/MDAnalysis/mdanalysis/pull/3566#discussion_r830414316), fixture for a single test isn't worth it.